.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+To help us get this pull request reviewed and merged quickly, please be sure to include the following items:
+
+* [ ] Tests (if applicable)
+* [ ] Documentation (if applicable)
+* [ ] A full explanation here in the PR description of the work done
+
+## PR Type
+What kind of change does this PR introduce?
+
+* [ ] Bugfix
+* [ ] Feature
+* [ ] Code style update (formatting, local variables)
+* [ ] Refactoring (no functional changes, no api changes)
+* [ ] Build related changes
+* [ ] CI related changes
+* [ ] Documentation content changes
+* [ ] Tests
+* [ ] Other
```

Comment:
Do we need `Other` here? What are we missing?

Reply:
maybe we didn't miss anything, but doesn't hurt to have it just in case? e.g adding code owners or something

Reply:
we may have to revisit, if there are many prs specified as **Other**